test/e2e/medusa_test.go

```diff
@@ -36,6 +36,22 @@ func createSingleMedusa(t *testing.T, ctx context.Context, namespace string, f *
 	verifyRestoreFinished(t, ctx, f, dcKey, backupKey)
 }
 
+func createSingleMedusaJob(t *testing.T, ctx context.Context, namespace string, f *framework.E2eFramework) {
```

Comment:
I realize that the setup will be a bit more involved that existing tests, but we should have a e2e test that covers remote restores. Would you create a follow up ticket for it?

Reply:
Sure thing

Reply:
https://github.com/k8ssandra/k8ssandra-operator/issues/542